---
aliases:
- awards-us
categories:
- documents
layout: page
description: Awards of Society for Open, Reliable, and Transparent Ecology and Evolutionary biology (SORTEE)
title: "SORTEE awards"
---

-   [Current Awards]({{< ref "awards.md#current-awards" >}})  
-   [Past Awards Winners]({{< ref "awards.md#past-awards-winners" >}})
-   [Past Awards Descriptions]({{< ref "awards.md#past-awards-descriptions" >}})
-   [TOP Publication Assessment Rubric]({{< ref "awards.md#top-rubric-for-publication-assessment" >}})

   

------------------------------------------------------------------------

 

# Current Awards

 

## SORTEE Commendation Awards

 

#### Award Description

-   The SORTEE Commendation Awards aim to highlight projects that align with the **aims and values of the society.**
-   A Project is defined as **any work** that is an initiative benefiting the Eco-Evo community by fostering Open Science values or practices (e.g., journal clubs, workshops, seminar series, websites, repositories, tools, educational materials, public talks, podcasts, etc.).
-   The SORTEE Commendation Awards recognise that Projects are often team efforts. As such, the contributions of all project members, rather than individuals, will be highlighted. Note, however, that projects from single individuals can also be nominated.
-   We warmly encourage **diverse nominations** in terms of team membership (e.g., country, gender, professional background) and content (see eligibility criteria below). We will also value projects with any scale of impact - from local (e.g., within a single research group or institution) to global (e.g., publicly accessible websites or repositories).
-   There is **no limit** on the number of SORTEE Commendation awards given by the society per year. Any nominated Projects meeting the eligibility criteria listed below will be awarded a Commendation.

 

#### Eligibility

1.  Projects must contribute to fostering Open Science values or practices such as transparency, robustness of research, or improvement of the scientific system/practice in general, going beyond simply implementing current practices or processes, with actual impact on any scale. Open-access data/code from journal publications is not considered going beyond current practices.
2.  Projects should be related at least remotely to the fields of ecology and/or evolutionary biology or closely related fields.
3.  Evidence of the project has to be provided, e.g., by photos, websites, teaching/outreach materials, screenshots of news or social posts, recommendation letters, repositories, preprints.
4.  The person nominating the project, or at least one project member, must be a current SORTEE member (newly joined members are eligible).
5.  Current SORTEE Awards Committee members, SORTEE Awards DEI Representative, members of the Board of Directors and Executive Officers ([the secretary, treasurer, past president, president, president-elect](sortee.org/people)) are not eligible to nominate a Project, or be a member of a nominated Project.

 

#### Nomination process

Project nominations can be submitted year-round until **two months before the SORTEE conference.** This two-month time period allows for assessing nominations by the Awards Committee and approval by the Board of Directors. Submissions made within this period will be transferred to the pool of submissions for the following round.

Projects may be self-nominated by project members, or nominated by individuals who were not project members. Project nominations must be submitted through an online [SORTEE Project Award nomination form](https://forms.gle/gFM47pAYe6mjECUU9) and a [document upload template.](https://docs.google.com/document/d/19R1tAcS_fNoXbxKvx3eaFYMhXJI7hvau/edit?usp=sharing&ouid=110356474356884735091&rtpof=true&sd=true) The nomination form requires the following information:

1.  Name of the nominated Project
2.  Name and email address of the person nominating the Project, and information on whether it is a self-nomination (from a project member) or not (from a person who is not a project member).
3.  Name of one member of the nominated Project, or the name of the nominating person, who is a current SORTEE member (there is no time limit between joining SORTEE and submitting an award nomination), for eligibility purposes.
4.  One completed nomination document (using the document upload template), containing the name of the nominated project and the following 3 sections:
    -   Names, email addresses, primary institutional affiliations (where appropriate), and contributions of all project members in a structured format as an attached document (file). We remind the nominator to ensure all under-represented groups are given appropriate credit for their contributions.
    -   Description (maximum 500 words) of why the Project is deserving of a SORTEE Commendation Award. This section may include showing how the project goes beyond simply implementing current scientific practices or processes, and its actual impact on any scale.
    -   Evidence supporting the project (maximum 2 pages). This section can contain copies of photos, websites, teaching/outreach materials, news and social posts, recommendation letters, repositories, preprints, or links (accessible to adjudicators through URLs / DOIs) to any additional documents evidencing the project.

 

#### Assessment process

-   The SORTEE Awards Committee will assess whether applications meet the SORTEE Commendation Awards eligibility criteria, as listed above.
-   Eligible Projects will then be proposed by the SORTEE Awards Committee to the SORTEE Board of Directors who will vote on the final approval.
-   We will inform the nominating person and project members about the assessment outcome using the contact details provided on the nomination form.
-   Awards will be announced during the SORTEE annual virtual conference and then posted on the [SORTEE website](http://www.sortee.org) and social media (currently: Mastodon, X, Facebook, LinkedIn).

 

[**NOMINATION DOCUMENT TEMPLATE**](https://docs.google.com/document/d/19R1tAcS_fNoXbxKvx3eaFYMhXJI7hvau/edit?usp=sharing&ouid=110356474356884735091&rtpof=true&sd=true) **(to be downloaded, filled in and provided as an attachment with the online submission form)**

 

------------------------------------------------------------------------

 

# Past Awards Winners

## **2025**

We are pleased to share the 2025 SORTEE Commendation Award recipients (listed in alphabetical order): EcoEvoApps, Flipping the Script, JuliaPhylo, MacaqueNet, MorphoBank, and the Virtual Australian Museum of Palaeontology (VAMP). You can learn more about each project in descriptions below.

### **EcoEvoApps**

**Website**: <https://ecoevoapps.gitlab.io/> 

**About**: Mathematical models are useful tools to study ecological and evolutionary dynamics. These models are often taught in undergraduate classes, but can appear intimidating to students, especially those without much quantitative training. EcoEvoApps addresses this gap with a collection of free Shiny apps and an R package that allow users to interactively explore mathematical models of ecology and evolution. By adjusting parameter values, users can instantaneously visualize the corresponding changes in model dynamics, supplementing other ways of learning these models. The platform currently hosts fourteen apps covering population ecology, species interactions, landscape ecology, and other specialized topics, and the collection continues to grow. Most of the apps are also available in Chinese, Portuguese, Spanish, and Turkish, in addition to English. Designed to lower barriers to quantitative learning, it supports users with diverse linguistic, cultural, and computational backgrounds. EcoEvoApps is a living and evolving project. It is fully open-source and community-driven, always welcoming general feedback, new app suggestions, collaborations in app development, and classroom use by educators and learners worldwide.

  

### **Flipping the Script**

**Website**: Lessons are hosted openly on QUBESHub (<https://qubeshub.org>)

**About**: This project transforms undergraduate biology education by embedding authentic data science training within an Ecology Lab course. Recognizing limited student exposure to big data and R programming, the instructional team created interactive R tutorials using the 'swirl' platform. These custom lessons arewoven into each lab module, allowing students to practice real-world statistical analysis with authentic ecological datasets. The interactive interface provides immediate feedback and adaptive hints, helping students "learn R in R" while reducing coding anxiety. Implemented through a flipped-class model, students complete coding prelabs before class, then build on their scripts during in-lab data analysis. This structure maximizes class time for experimentation and reinforces transparent, reproducible research practices. By independently coding, students see both the creativity and pitfalls of data analysis, fostering appreciation for meticulous scientific documentation. The project has increased student confidence, engagement, and identity as scientists, while also expanding departmental teaching capacity. Graduate and undergraduate TAs can now effectively assist students while gaining programming skills themselves. Ecology Lab enrollment has doubled, and the course now serves as a core course for a new Environmental Science major. The materials are freely available through the NSF-supported QUBES platform, supporting national reform in quantitative biology education.

  

### **JuliaPhylo: a software ecosystem for phylogenetics in Julia**


![](/img/award_juliaph.png)


**Website**: <https://juliaphylo.github.io/JuliaPhyloWebsite/> 

**About**: The JuliaPhylo project grew from small beginnings into an ecosystem of Julia packages for the inference, manipulation, visualization, and use of evolutionary networks to study trait evolution. Evolutionary networks, also referred to as 'phylogenies', include trees (phylogenetic gene/species trees) and phylogenetic networks as well as admixture graphs. Networks can encode hybrid speciation events, gene flow between species, and admixture between populations. We chose the Julia programming language because it is open-source, and combines ease-of-use with speed. JuliaPhylo packages now receive contributions from developers across the research community. JuliaPhylo is modular, with different features and methods living in different packages. Some packages focus on core utilities to manipulate trees and networks, to plot them, or to summarize a sample of phylogenies. Other packages focus on inferring phylogenies. Others focus on evolutionary simulations: simulating gene trees within population / species phylogenies (under the multispecies coalescent), or simulating traits controlled by these genes. Finally, some packages focus on analyzing trait evolution along phylogenies. This modularity, and being open-source, makes it more accessible for potential contributors to join the project and collaborate on implementing new features. This growth is exciting!

  

### **MacaqueNet**


![](/img/award_macaquenet.png)

**Website**: <https://macaquenet.github.io/>  

**About**: MacaqueNet is a global collaborative community hosting the largest standardised, publicly searchable database of animal social behaviour. It contains data on over 3,000 individual macaques from 61 populations spanning 14 species, contributed by 118 researchers across 58 institutions in 18 countries. This community-driven initiative is dedicated to making data more accessible, standardised, and reusable. Developed to overcome challenges with data sharing, including inconsistent methods hindering standardisation and concerns over misinterpretation, MacaqueNet works with contributors to archive raw data and metadata using well-defined protocols for standardisation, archiving, and sharing. Data sharing follows FAIR (Findable, Accessible, Interoperable, Reusable) principles under community-agreed terms of use. Nearly 700 social networks are centralised and requestable, allowing researchers to conduct large-scale comparative analyses that would be impossible using single or limited populations. Beyond the database, MacaqueNet connects researchers worldwide through biennial symposia, an active website, a Slack workspace, and six-monthly newsletters. It supports early-career scientists and researchers from underrepresented regions, and those without access to long-term datasets. With multiple cross-species projects already using the database, MacaqueNet is helping drive open, inclusive, collaborative science and contributing to greater transparency, reproducibility, and data reuse---efforts that will help strengthen research across behavioural ecology and evolutionary biology. You can sign-in for MacaqueNet's newsletter [here](https://docs.google.com/forms/d/e/1FAIpQLSef2OLRBd0Bd4PcgsASWWF8JEb2ZE%20L90XugA2lqLlRud7qDGQ/viewform).

  

 

  

### **MorphoBank**

![](/img/award_morphobank.png)

**Website**: <https://www.morphobank.org/> 

**About**: MorphoBank is an open-access, FAIR-aligned research repository designed to support evolutionary research based on phenotypes (morphology), particularly in systematics, cladistics, and phylogenetics. Because phenotype-based research relies on explicitly defined characters and character states, often supported by images and other media, MorphoBank provides purpose-built tools through an integrated web application and database that enable scientists to collect, curate, annotate, and publish datasets in a structured and reusable way. Developed with support from the U.S. National Science Foundation and sustained by institutional and community partners, MorphoBank facilitates the open sharing of peer-reviewed observations as both text and 2D and 3D images. Beyond serving as a repository, MorphoBank enables real-time collaboration among distributed research teams, allowing scientists to jointly build and refine phylogenetic matrices and directly link character statements to supporting media. Dedicated staff curation improves metadata quality and long-term usability, ensuring that datasets remain findable, interpretable, and reusable over time. As of December 12, 2025, MorphoBank hosts 2,043 publicly accessible projects, comprising 1,426 matrices and 211,111 media files, with an additional 2,176 projects in progress that will be released as studies are completed. The platform supports 4,658 scientists and students as active content builders, and more than 21,500 site visitors viewed or downloaded data in the past 30 days. Together, these efforts position MorphoBank as a shared, community-driven infrastructure advancing open, transparent, and reproducible eco-evo science. You can follow MorphoBank's updates on [Bluesky](https://bsky.app/profile/morphobank.bsky.social) and [LinkedIn](https://www.linkedin.com/company/phoenix-bioinformatics/).

  

###  **Virtual Australian Museum of Palaeontology (VAMP)**

![](/img/award_VAMP.png)
**Website**: <https://sites.flinders.edu.au/vamp/> 

**About**: VAMP is an online, open access virtual museum dedicated to showcasing Australia's fossil heritage in 3 D. Through VAMP, researchers, educators and the general public can explore high resolution, interactive, downloadable 3D scans of fossil specimens, from some of the earliest complex life to Australia's iconic extinct megafauna. VAMP brings together fossils held in museum collections, many of which are rare or "type specimens" not usually accessible to the public. We are a collaboration led by early and mid career researchers at Flinders University in partnership with institutional partners South Australian Museum (SAM), Western Australian Museum (WAM), and Museum and Art Gallery of the Northern Territory (MAGNT) and others. Using modern scanning technologies (such as CT and micro CT scans, synchrotron/neutron tomography, photogrammetry, LiDAR, and surface scanning) , the project digitises fossils without damaging them. This approach allows detailed study of both external shapes and internal structure, enabling new kinds of research, outreach, teaching, and public engagement. From early Ediacaran life to Pleistocene megafauna, VAMP provides a window into 600 million years of Australian palaeontological heritage, Celebrating digital Australian fossils in 3D and providing a valuable resource for scientists, students, and curious minds alike.   


## **2024**

In 2024, we focused on revising and improving the awards criteria and therefore did not present any awards that year. 

## 2023

**2023 Open Science Student Award**  
![Euan Angus Young](/img/EuanYoung.png)

### *Name*: Euan Angus Young

*Pronouns*: he/him  
*Affiliation*: University of Groningen, The Netherlands  
*Twitter*: \@euantheyoung  
*Profile*: I am interested in how an evolutionary perspective can inform our understanding of human health and disease, and how humans themselves can be used as a model for understanding evolutionary processes. To these ends, I use a data-driven approach that exploits both historical and contemporary sources. When embarking upon such research, you become aware of how many decisions must be made after data collection, but before results are reported and it is vital that we are transparent with this process of data handling and analysis. Of course, these details would be included in any published article, but with the sharing of code and data we can allow the verification of these steps with more transparency. This gives others the opportunity to check our work directly and replicate, if desired, making open science practises an essential tool for increasing the transparency and robustness of scientific findings.

 

**2023 Open Science Researcher Award**  
![Birgit Szabo](/img/BigitSzabo.png)

### *Name*: Birgit Szabo

*Pronouns*: they/them  
*Twitter*: \@birgit_szabo  
*Affiliation*: Division of Behavioural Ecology, Institute of Ecology and Evolution, University of Bern, Switzerland  
*Profile*: I consider myself a cognitive ecologist studying the link between ecology, cognition and behaviour in my chosen model system - lizards. I use lizards as models because they express diverse ecology including sociality. By studying the consequences of social life on cognition, social behaviour and stress coping we can better understand how social life influences these traits that might help individuals strive in a social environment. In my research, I am dedicated to research integrity which makes open science an integral part of my work. I have incorporated open science practices into my workflow so I can publish preprints, data and code without much additional effort, which is key in the busy schedule of research. I have also been luck to publish much of my work open access. Even though I put much effort into open science practices, I want to continue improving hopefully with my first preregistration in the near future. As a supervisor and mentor I also strive to help students and colleagues understand the importance of open science making some progress in changing people's minds about open science. I am convinced that open science will help us produce better science in the future.

   
 

## 2022

**2022 Open Science in Practice PhD Student Award**  
![Patrice Pottier](/img/PatricePottier.png)

### *Name*: Patrice Pottier

*Pronouns*: he/him/his  
*Twitter*: \@PatriceEcoEvo  
*Website*: [patricepottier.owlstown.net](patricepottier.owlstown.net)  
*Affiliation*: The University of New South Wales, Sydney, Australia  
*Profile*: I am a Ph.D. Candidate in the Evolution & Ecology Research Centre at UNSW Sydney. While my interests are broad, I am primarily interested in the responses of animals to rapid environmental change. Particularly, I strive to understand what drives the variation in plasticity and adaptation to changing temperatures. My research is question-driven rather than organism-driven, and I enjoy working with a broad range of organisms. I use a combination of data synthesis, meta-analysis, and laboratory experiments to address my questions in various species. I am also an advocate for Open Science and a member of the board of directors of The Society for Open, Reproducible, and Transparent Ecology and Evolutionary Biology (SORTEE). Outside of academia, I love surfing, traveling, and eating all the vegan food the world has to offer. \[more at: <https://peerj.com/blog/post/115284886214/peerj-award-winners-sortee-2022/>\]

 

**2022 Open Science in Practice Postdoctoral Award**  
![Korryn Bodner](/img/KorrynBodner.png)

### *Name*: Korryn Bodner

*Pronouns*: she/her/hers  
*Affiliation*: Unity Health Toronto, Canada  
*Twitter*: \@KorrynBodner  
*Profile*: My main research focus is on the epidemiology of infectious diseases. I primarily study emerging SARS-CoV-2 Variants of Concern (VOCs) and how heterogeneities in contact patterns and vaccination strategies can affect transmission dynamics.  Through this work, I have also become engaged in studying the effects of different types of biases that can undermine the results of our mathematical and statistical models. As an example, recently I have explored how unknown and underlying differences in contact and testing patterns between vaccinated and unvaccinated individuals can lead to an underestimate of the benefits of a given vaccine. More broadly, I am also interested in developing strategies to increase reproducibility and to better connect scientific research with decision-making. This area of focus is particularly rewarding as its utility extends beyond any particular model, ecological scale, or specific research purpose. \[from (and more at): <https://peerj.com/blog/post/115284886214/peerj-award-winners-sortee-2022/>\]

 

**2022 Promotion of Open Science Award**  
![Silas Bossert](/img/SilasBossert.png)

### *Name*: Silas Bossert

*Affiliation*: Research Entomologist at Washington State University  
*Profile*: I am an evolutionary biologist studying the phylogeny and evolutionary history of bees. I am fascinated by the parallel evolution of bees and plants over the past 125 million years and by the many ways bees live. Through the phylogenetic lens, I strive to understand the natural history of bee-plant interactions, their antiquity, and the evolution of parasitism in bees. We know of over 20,000 described species of bees and many more await to be discovered. \[from (and more at): <https://peerj.com/blog/post/115284886214/peerj-award-winners-sortee-2022/>\]

   
 

## 2021

**2021 Open Science in Practice Masters Student Award**  
![Shivani](/img/Shivani.png)

### *Name*: Shivani

*Pronouns*: she/her/hers  
*Affiliation*: University of Göttingen, Germany  
*Profile*: My broad research interests are in Behavioral Ecology and Evolutionary Biology. Mainly, I work with Primates and try to understand the different forces that shape their behavior. I am keen on using interdisciplinary approaches to understand the behavior of different life forms and aim to keep my research as open and transparent as possible. Currently, for my PhD research, I look at the social and ecological drivers of female fitness in a wild population of Assamese macaques.

 

**2021 Open Science in Practice PhD Student Award**  
![Nicole Torres-Tamayo](/img/NicoleTT.png)

### *Name*: Nicole Torres-Tamayo

*Pronouns*: she/her/hers  
*Affiliation*: School of Life and Health Sciences, University of Roehampton, London, United Kingdom  
*Twitter*: \@Paleonicole  
*Profile*: Nicole Torres-Tamayo is a post-doctoral researcher at the School of Life and Health Sciences of the University of Roehampton (London, United Kingdom). She investigates the evolution of human childbirth by studying the pelvis morphology of extant primates. She also has a background in Paleoanthropology and she is currently working on the reconstruction of the fragmentary fossil pelvis of H. erectus and H. neanderthalensis using 3D virtual techniques. During her early career and with her PhD impacted by the COVID-19 pandemic, Nicole became aware of the importance of data sharing and open science in research. This is paramount in the field of Paleoanthropology, where there are too few fossils from which people draw too many conclusions, sometimes without allowing others to check the actual remains. Her next step as a post-doctoral researcher is the Institute of Evolutionary Medicine of the University of Zurich, where she will continue actively working on promoting transparency and reproducibility in science. 

 

**2021 Open Science in Practice Postdoctoral Award**  
![Benjamin H. Paffhausen](/img/Benjamin_H_Paffhausen.png)

### *Name*: Benjamin H. Paffhausen

*Affiliation*: Centre de Recherches sur la Cognition Animale, Centre de Biologie Intégrative (CBI), University of Toulouse, CNRS, UPS, Toulouse, France  
*Profile*: I want to understand the social communication in the honey bees brain during waggle dance. I started when the open source microcontroller Arduino was becoming popular. The code copying mindset got me and I immediately understood that sharing openly is the only way forward. I investigated color vision in honey bees with an Arduino driven illumination arm. This gave us great data sets that were sufficiently repetitive to analyse across animals. I took the servo-motor library and wrote a bit of code for the machine to scan along. Inaccessible for me before open source, but with it: a joy. Later we put a bee on a quad-copter while we recorded brain activity to investigate the animal's navigation. The electrical engineers of reddit shared with me how to design the electronics board. Earlier I recorded bees' brain activity in their hive to investigate free and motivated social behaviour, impossible without the collaborative work that was done on the Raspberry Pi to monitor my bees. Those and other projects of mine focus on the peculiar behaviour of the honey bee and quickly accessing them by taking useful snippets from others. All works were acknowledged and given back.

 

------------------------------------------------------------------------

 

# Past Awards Descriptions

## 2023 Open Science Student and Open Science Researcher

This award aims to recognize and reward researchers who have made consistent and long term contributions to open science in various forms including publications, preregistration, data or software sharing, and outreach (e.g. the presentation of seminars and workshops), benefitting broadly defined ecology and evolutionary biology.

Two separate awards were available in 2023: one for currently registered students, and one for researchers NOT currently registered as students.

**Application materials:**

1.  A brief description (maximum 500 words) of why you (or your nominee) are deserving of this award, and referring to evidence and examples (accessible through URLs / DOIs) of how you have integrated ORT practices into your research workflow as a student researcher. Also mention how your work is relevant to ecology or evolutionary biology. You should primarily highlight your own (or your nominee's) ORT contributions while clearly identifying contributions from supervisors and collaborators.
2.  A brief description (maximum 500 words) of other elements of ORT that you wish you (or your nominee) had applied to the research, but were limited by financial constraints, access to certain resources, bureaucratic restrictions, or any other barriers.
3.  Upload of any relevant additional files, for example, an article you are particularly proud of, or relevant materials not available online. Please do not upload recommendation letters or CV.
4.  Provide name, affiliation, and email of potential Referee 1 and Referee 2.

 

## 2021 - 2022 Open Science in Promotion

This award aims to recognize and reward researchers who have made consistent and long term contributions to open science in various forms including publications, data sharing, and outreach (e.g. the presentation of seminars and workshops).

Two separate awards were available in 2022: one for early career (5 or fewer years post-PhD) and one for established (\>5 years post-PhD) researchers.

**Application materials:**

\(i\) A letter (maximum 500 words) describing why you are deserving of this award, and referring to evidence and examples (accessible to adjudicators through URLs / DOIs) of how your publication and the work that went into it exemplifies Open Science best practices and the promotion of Open Science.

\(ii\) An up-to-date resumé (maximum 2 pages as a .pdf)

\(iii\) A list with explanations of open science contributions

 

## 2021 - 2022 Open Science in Practice

This award aims to recognize and reward researchers who have endeavoured to implement best practices in Open Science (OS) within their research workflow, thereby increasing the transparency and reproducibility of their research activities.

Three separate awards are available each year: one for Masters students (current, or \<12 months since degree conferred), one for PhD students (current, or \<12 months since degree conferred), and one for postdoctoral research associates (\< three years since doctoral degree conferred).

All steps that are taken to improve the transparency and reproducibility of research are valuable and appreciated, and achieving the "ideal" workflow in practice will, for many of us, be a life-long endeavour and learning experience! We therefore encourage all eligible researchers to apply, regardless of how far along this path they are. For example, a graduate student or postdoc may not have implemented many or any OS best practices from the outset of their research career because they initially did not have access to the training and support necessary. Thus, their earlier work may be less transparent and reproducible than their later work. This is entirely expected, and should not discourage you from applying for this award.

**Application materials:**

\(i\) A letter (maximum 500 words) describing why you are deserving of this award, and referring to evidence and examples (accessible to adjudicators through URLs / DOIs) of how you have integrated OS best practices into your research workflow as a graduate student or postdoc.

\(ii\) An up-to-date resumé (maximum 2 pages).

\(iii\) A brief document (maximum 300 words) describing other elements of reproducibility or open science that you wish to have applied to your research, but were limited by financial constraints, access to certain resources, bureaucratic restrictions, or any other barrier.

**Criteria:**  
The criteria for judging submissions are based on [TOP journal guidelines](https://www.cos.io/initiatives/top-guidelines) modified for application to individual publications. We emphasize that making publications Open Access is NOT a criterion for the awards, as this typically requires access to substantial funds. However, making a penultimate version of an officially accepted manuscript publicly available (e.g. on a preprint server, institutional repository, or personal webpage) is good practice, and is now permitted by almost all publishing outlets.

 

------------------------------------------------------------------------

 

# TOP Rubric for Publication Assessment

*(1 point for each checkbox)*

1.  Data, Analytical Methods, Code, and Research Materials Transparency  

-   Did the author (hereafter applicant) indicate that the data, methods used in the analysis, and materials used to conduct the research are publicly available.  
-   Did the applicant make the data available at a trusted digital repository? (Note: If all data required to reproduce the reported analyses appears in the article text, tables, and figures then it does not also need to be posted to a repository.)  
-   Did the applicant include all variables, treatment conditions, and observations described in the manuscript?  
-   Did the applicant provide a full account of the procedures used to collect, preprocess, clean, or generate the data?  
-   Did the applicant provide program code, scripts, codebooks, and other documentation sufficient to precisely reproduce all published results?  
-   Did the applicant provide research materials and description of procedures necessary to conduct an independent replication of the research?

1.  Design and Analysis Transparency  

-   Did the applicant report on the process by which they followed standards for disclosing key aspects of the research design and data analysis. For example, did the applicant review the standards available for many research applications from <http://www.equator-network.org/> and use those that are relevant for the reported research applications?

1.  Preregistration of Study  

-   Did the applicant, in acknowledgments or the first footnote, indicate if they did or did not pre-register the research in an independent, institutional registry?  
-   If an applicant did preregister the research, the applicant must confirm that the study was registered prior to conducting the research with links to the time-stamped pre-registrations at the institutional registry, and that the preregistration adheres to the disclosure requirements of the institutional registry or those required for the preregistered badge maintained by the Center for Open Science.  
-   Did the applicant report all pre-registered analyses in the text, or, if there were changes in the analysis plan following preregistration, those changes must be disclosed with explanation for the changes?  
-   Did the applicant clearly distinguish in text analyses that were preregistered from those that were not, such as having separate sections in the results for confirmatory and exploratory analyses?

   
